<launch>
   
    <arg name="K_eu" default="0.5" />
    <arg name="K_ev" default="0.7" /> 
    <arg name="K_rho" default="0.5" />
    <arg name="K_alpha" default="0.5" />  
    <arg name="K_beta" default="0.5" />
    <arg name="K_int" default="0.25" />     
    <arg name="threshold" default="0.3"/>
    <arg name="max_lin" default="0.5" />
    <arg name="max_ang" default="0.5" />   
    <arg name="control_type" default="2"/>
    <arg name="camera_height" default="0.1"/> 
   

 	<arg name="path" default="$(find servo_visao)/cfg/"/>
   
    <node name="visual_servoing" 
        pkg="servo_visao" 
        type="turtlebot3_visual_servoing_pbvs_v2.py"
        args="$(arg K_eu) $(arg K_ev) $(arg K_rho) $(arg K_alpha) $(arg K_beta) $(arg K_int) $(arg threshold)
            $(arg max_lin) $(arg max_ang) $(arg control_type) $(arg camera_height)"
        output="screen">

	<remap from="img_point" to="/camera/img_base"/>
	<remap from="odom" to="/odom"/>
	<remap from="cmd_vel" to="/cmd_vel"/>
	<remap from="control_type" to="/control_type"/>
	<remap from="camera_info" to="/camera/rgb/camera_info"/>
	</node>
	
</launch>
